container_pipeline/scanners/misc_package_updates.py

```diff
@@ -59,7 +59,9 @@ def process_output(self, logs):
         """
         data = {}
         data["scanner_name"] = self.scanner_name
-        data["msg"] = "Results for miscellaneous package manager updates"
+        data["msg"] = ""
+        for i in logs:
+            data["msg"] += i["Summary"]
```

Comment:
will here new lines be needed ?

Reply:
No. We're not adding newlines. We're adding space to have a single line message based on summaries from pip, npm and gem update check.